lib/ai/unified-summary-parser.ts

```
@@ -69,19 +90,69 @@ export function parseUnifiedResponse(text: string): ParsedSummaryResult {
           }
           break;
         case 'detailed':
-          // 詳細要約の内容を収集（改善版）
+          // 詳細要約の内容を収集（改善版：カテゴリ削除 + 1行連結 + プロンプトフィルタ）
           if (trimmed.startsWith('・') || trimmed.startsWith('-') || trimmed.startsWith('*')) {
-            detailedSummaryLines.push(trimmed);
+            // プロンプト行をフィルタリング
+            const isInstructionDetailed = INSTRUCTION_PATTERNS.some(pattern => pattern.test(trimmed));
+            if (!isInstructionDetailed) {
+              // 「・カテゴリ：項目名」または「・項目名：内容」形式を検出
+              const match = trimmed.match(/^[・-]\s*(.+?)[:：]\s*(.*)$/);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)